src/styles/Home.css

```diff
@@ -0,0 +1,108 @@
+@media only screen and (max-width: 767px) {
+  .home-container {
+    background-color: rgba(205, 44, 125, 0.819);
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .header {
+    display: flex;
+  }
+
+  .header-img {
+    width: 100%;
+    border: 3px solid aliceblue;
+    opacity: 0.8;
+  }
+
+  .headline {
+    text-decoration: underline;
+    text-underline-offset: 10px;
+    color: aliceblue;
+  }
+
+  .search-bar-wrapper {
+    width: 100%;
+    margin: 0 auto;
+  }
+
+  input[type="text"] {
+    width: 100%;
+    padding: 10px;
+    font-size: 16px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+  }
+
+  .table-wrapper {
+    margin-top: 10px;
+  }
+
+  .Breeds {
+    color: purple;
+    margin: 0;
+    padding: 0;
+    align-self: flex-start;
+  }
+
+  .temp {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .table {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 0;
+    margin: 0%;
+    justify-content: center;
+  }
```

Comment:
- [ ] Well done, by implementing the changes asked by your previous code reviewer. It seems like you did work but not completely. Your design is still not matching the wireframe. Kindly fix the blank spaces around the cards, and also make the background colour of each card different as given in the original design. It is a capstone project and you should consider every detail of the project in order to get a successful developer. Make sure that your design follows the original design.

### Design Comparison
<table>
<tr>
<th>Behance Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/Rafiduzzaman/react-capstone/assets/66476329/43c22e8d-f8b7-4960-bcce-2e3d92f83ef3)

</td>
<td>

![image](https://github.com/Rafiduzzaman/react-capstone/assets/66476329/2d8659df-8514-4cb1-83fe-e2efe7ca8f0c)

</td>
</tr>
</table>